CMakeLists.txt

```diff
@@ -56,11 +56,7 @@ if(MSVC)
     # ROOT_BINDIR is set by ROOTConfig.cmake
     get_filename_component(realfp "${ROOT_BINDIR}/../" ABSOLUTE)
     file(TO_CMAKE_PATH ${realfp} ROOTSYS)
-    # FIXME: Get from root-config.bat --tutdir
```

Comment:
Don't we want to keep this as a reminder?

Reply:
I tried, but `root-config.bat --tutdir` returns just `tutorials` in buit-in mode on Windows...

Reply:
But shouldn't that be fixed instead? Does it work with the stand-alone build?

Reply:
Yes, it works in stand-alone. I'll check what's different between Linux and Windows...

Reply:
But it shouldn't really matter anyway (it's  semantically the same)

Reply:
@hahnjo I'm fixing `root-config.bat.in` right now. Will fix roottest afterwards